src/ucp/wireup/wireup.h

```diff
@@ -58,6 +58,7 @@ struct ucp_wireup_tl_info {
     uint64_t                   cap_flags;  /* Interface capability flags */
     float                      overhead;   /* Interface performance - overhead */
     float                      bandwidth;  /* Interface performance - bandwidth */
+    int                        priority;   /* Priority of device */
```

Comment:
let's make it uint8_t (and the structure - packed) so the address size will not be enlarged by much